erc/ERCS/erc-7562.md

```diff
@@ -238,9 +330,8 @@ This means that a `Paymaster` and `Factory` contracts cannot practically be an "
     * Do not add a `UserOperation` to the mempool if the maximum total gas usage, including the new `UserOperation`, is above the deposit of the `paymaster` at the current gas price.
     * TODO: allow "fractional reserve" based on reputation?
```

Comment:
We can change the rule, and allow staked paymaster any # of userops.
It does require tuning the opsSeen/opsIncluded rules, so that such paymasters are only throttled (not banned) on overload.
but it is complex, and I wanted to keep the basic doc simpler.

Reply:
Right, it's best to keep the doc simpler if possible and not change the way reputation works. Should we still allow some over-commitment ratio for staked entities?  Or just leave it at 1:1?

In any case this is a "local" rule so bundlers may choose to manage their own ratio and ignore this rule if they know what they're doing.  That's why it's important to mark local rules as such.